### ISSUE TITLE
Implementation: VideoFile Type

### ISSUE THREAD

**Issue by lafemmephile** [lafemmephile/core#20]
Would be good to do some custom *type checking* on all the commands using a specific *video path*.

Possible design:
```python
class VideoFile(click.Path):
    # check video file extension is appropriate ...
```

**Comment by lafemmephile** [comment]
Where to implement:
+ https://github.com/lafemmephile/core/blob/6145441e64e95a118751422d670916e8e47e4fd6/src/femmephile/__main__.py#L116-L121
+ https://github.com/lafemmephile/core/blob/6145441e64e95a118751422d670916e8e47e4fd6/src/femmephile/__main__.py#L172-L175 
+ https://github.com/lafemmephile/core/blob/6145441e64e95a118751422d670916e8e47e4fd6/src/femmephile/__main__.py#L254-L257

Basically everywhere the `<video_path>` argument is used

**Comment by lafemmephile** [comment]
Implemented with: 6dee15767d127478bdb37c8013754b695a61006a
